setup.py

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python3
 
-# Start ignoring PyImportSortBear as imports below may yield syntax errors
+# Start ignoring PyImportSortBear and PycodestyleBear* as imports below may
+# yield syntax errors
 from bears import assert_supported_version
 assert_supported_version()
```

Comment:
Actually, there are a lot of assumptions here.
Firstly, the test wasnt even working: https://github.com/coala/coala/issues/3310
Secondly, I've checked the package `__init__.py` for coala and -bears, and neither drags in lots of dependencies, so we should be ok to import the package early in the process.  I've not finished analysis of this part though.